_includes/components/hero.html

```diff
@@ -21,7 +21,7 @@ <h2>
       {{ hero.content | markdownify }}
 
       {% if hero.button %}
-      <a class="usa-button usa-button-big usa-button-secondary"
+      <a class="usa-button usa-button-big"
```

Comment:
It looks like `usa-button-big` has also been removed in the latest hero component: https://components.standards.usa.gov/components/detail/hero.html